frontend/src/Pages/DebateRoom.tsx

```
@@ -7,6 +7,11 @@ import JudgmentPopup from "@/components/JudgementPopup";
 import { Mic, MicOff } from "lucide-react";
 import { useAtom } from "jotai";
 import { userAtom } from "@/state/userAtom";
+import { useNavigate } from "react-router-dom";
```

Biome (2.1.2)

[error] Shouldn't redeclare 'useNavigate'. Consider to delete it or rename it.

'useNavigate' is defined here:


(lint/suspicious/noRedeclare)